src/libs/modulegroups.c

```diff
@@ -3763,6 +3823,11 @@ static void _manage_show_window(dt_lib_module_t *self)
   gtk_widget_set_name(d->basics_chkbox, "modulegroups_editor_setting");
   g_signal_connect(G_OBJECT(d->basics_chkbox), "toggled", G_CALLBACK(_manage_editor_basics_toggle), self);
   gtk_box_pack_start(GTK_BOX(vb), d->basics_chkbox, FALSE, TRUE, 0);
+  d->edit_full_active_cb = gtk_check_button_new_with_label(_("show all history in active group"));
```

Comment:
I would add a tooltip:
"the active group will show enabled and disabled modules"

(BTW, the active group in this scenario is no more "active group" but more "history group")

Reply:
Or "should modules in the history regardless of whether or not they are currently enabled"

Reply:
```suggestion
  d->edit_full_active_cb = gtk_check_button_new_with_label(_("show all history modules in active group"));
```